pyproject.toml

```diff
@@ -25,16 +25,21 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-    "xarray",
-    "rioxarray",
+    "aiobotocore==2.5.0",
+    "botocore==1.29.76",
+    "mangum>=0.10.0",
+    "netCDF4==1.6.2",
+    "xarray==0.19.0",
+    "rioxarray==0.14.1",
     "zarr",
-    "fsspec",
-    "s3fs",
+    "fsspec==2023.4.0",
+    "s3fs==2023.4.0",
```

Comment:
😬 do we really need all those dependencies? 

This adds a lot

Reply:
e.g we don't need to add 
```
    "aiobotocore==2.5.0",
     "botocore==1.29.76",
     "mangum>=0.10.0",
``` 

here because those dependency are only useful in AWS Lambda (which we install in the docker image)

Reply:
I'll see which ones we need, I want to make sure the dependencies installed locally are the same versions as get run in the docker image so I can hope that if my local version is running the docker version will also work. A few times now things are broken in the deployed instance when things are working locally, due to dependency differences.

Reply:
I think it's important to make the difference between what we use locally and what we have in the AWS Lambda environment. E.g the version for `s3fs` will be entirely defined by the version of botocore found in AWS Lambda which IMO is not something that would have impact on running the code locally but might mess with your local environment (if you're not using virtual env)

Reply:
I see what you are saying. I feel it is very useful to know how to run the application locally so that it simulates the deployed version and makes it easier to troubleshoot locally. As a middle ground, I have removed the extra dependencies from this file and write some about how to install AWS Lambda dependencies in the README. Let me know if this approach works for you.

Reply:
we could totally adds a `requirement-lambda.txt` if you want! (my original comments was that we shouldn't add things to `-dev` because it was use for deployment purpose)

Reply:
I'll refactor the whole `stack` directory to have something more like https://github.com/developmentseed/eoAPI/tree/main/infrastructure in another PR